proto/sifnode/dispensation/v1/types.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+package sifnode.dispensation.v1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/coin.proto";
+
+option go_package = "github.com/Sifchain/sifnode/x/dispensation/types";
+
+message GenesisState {
+  repeated string address_whitelist = 1;
+  string          admin_address     = 2;
+}
+
+// Distribution type enum
```

Comment:
We should remove these comments for DistributionType - they don't add any information

Reply:
Proto linters expect this so will be common place.